Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -1104,3 +1121,67 @@ def remove_clipping_plane(self):
 
             self.getRenderer().Render()
             self.updatePipeline()
+
+    def createAnimation(self,
+                        FrameCount=20,
+                        InitialCameraPosition=None,
+                        FocalPoint=None,
+                        ClippingRange=None,
+                        AngleRange=360,
+                        ViewUp=None):
+
+        viewer = self
+
+        if InitialCameraPosition is None:
+            InitialCameraPosition = viewer.getCamera().GetPosition()
+        if FocalPoint is None:
+            FocalPoint = viewer.getCamera().GetFocalPoint()
+        if ClippingRange is None:
+            ClippingRange = (0, 2000)
+        if ViewUp is None:
+            ViewUp = (0, 0, 1)
+        if FrameCount is None:
+            FrameCount = 100
+        #Setting locked values for camera position
+        locX = InitialCameraPosition[0]
+        locY = InitialCameraPosition[1]
+        locZ = InitialCameraPosition[2]
+
+        print('Initial Camera Position: {}'.format(InitialCameraPosition))
+        #Setting camera position
+        viewer.getCamera().SetPosition(InitialCameraPosition)
+        viewer.getCamera().SetFocalPoint(FocalPoint)
+
+        #Setting camera viewup
+        viewer.getCamera().SetViewUp(ViewUp)
+
+        #Set camera clipping range
+        viewer.getCamera().SetClippingRange(ClippingRange)
+
+        #Defining distance from camera to focal point
+        r = numpy.sqrt(((InitialCameraPosition[0] - FocalPoint[0])**2) + (InitialCameraPosition[1] - FocalPoint[1])**2)
+        print('Radius (distance from camera to focal point): {}'.format(r))
+
+        #Animating the camera
+        for x in range(FrameCount):
+            angle = (2 * numpy.pi) * (x / FrameCount)
+            NewLocationX = r * numpy.sin(angle) + FocalPoint[0]
+            NewLocationY = r * numpy.cos(angle) + FocalPoint[1]
+            NewLocation = (NewLocationX, NewLocationY, locZ)
+            camera = vtk.vtkCamera()
+            camera.SetFocalPoint(FocalPoint)
+            camera.SetViewUp(ViewUp)
+            camera.SetPosition(*NewLocation)
+            viewer.ren.SetActiveCamera(camera)
+            viewer.adjustCamera()
+
+            import time
+            time.sleep(0.05)
+            print("render frame {} angle {}".format(x, angle))
+            print('Camera Position: {}'.format(NewLocation))
+            rp = numpy.sqrt(((NewLocation[0] - FocalPoint[0])**2) + (NewLocation[1] - FocalPoint[1])**2)
+            print('Camera trajectory radius {}'.format(rp))
+            viewer.saveRender('test_{}'.format(x))
+            #Rendering and saving the render
+            viewer.getRenderer().Render()
+            viewer.renWin.Render()
```

Comment:
I suggest to create a new PR with this functionality. Please remove from this PR.